README.md

```diff
@@ -155,8 +156,8 @@ Tasks cleanup
 -------------
 
 Although, the history of tasks has an auditing value, some kinds of
-tasks can grow up in number quite soon. Therefore there is a mechanism
-how to clean the tasks, using a rake command. When running without
+tasks can rapidly increase. Therefore, there is a mechanism for
+how to clean the tasks which is done by using a rake command. When running without
```

Comment:
```suggestion
cleaning up the tasks which is done by using a rake command. When running without
```

Reply:
Hey @ezr-ondrej! I like your suggested change of, "cleaning up the tasks which is done by using a rake command. When running without" and think it does smooth out the sentence structure.